RELEASE/scripts/autoscend/iotms/mr2021.ash

```diff
@@ -112,3 +112,245 @@ int auto_backupUsesLeft()
 {
 	return 11 + (my_path() == "You, Robot" ? 5 : 0) - get_property("_backUpUses").to_int();
 }
+
+boolean auto_havePowerPlant()
+{
+	return possessEquipment($item[potted power plant]) && auto_is_valid($item[potted power plant]);
+}
+
+void auto_harvestBatteries()
+{
+	if (auto_havePowerPlant())

```

Comment:
usually what we do is
```
if(!auto_havePowerPlant())
{
  return false;
}
```
then do a bunch of stuff afterwards. that way we do not have huge unwieldy ifs that wrap around entire functions.
and it avoids confusion when we have several things to test

Reply:
Too easy to fix, thanks for the heads up!

Reply:
oh sorry. when I said use `return false` I meant just `return`.
I did not actually mean that you should change it to a boolean function.
this is fine though